src/doc/gui_manual/Preferences/File_Locations.rst

```diff
@@ -0,0 +1,297 @@
+* Location and file name(s): There are separate directories in ``VUSER_HOME``
+  for operator, database and plot plugins. These are
+
+  * ``VUSER_HOME/<visit-version>/<visit-arch>/plugins/operators/``
+  * ``VUSER_HOME/<visit-version>/<visit-arch>/plugins/databases/``
+  * ``VUSER_HOME/<visit-version>/<visit-arch>/plugins/plots/``
+
+  where ``<visit-version>`` and  ``<visit-arch>`` are the VisIt_ version number
+  and VisIt_ architecture moniker. A single plugin involves a set of related
+  files for the mdserver, engine and those common to both.
+* Purpose: Directories to hold custom (e.g. user-compiled and installed)
+  plugins.
+* Written: When the user
+  `makes and installs a custom plugin <http://visitusers.org/index.php?title=Building_plugins_using_CMake>`_
+* Read: On VisIt_ startup, all plugin *info* files are read. The remaining
+  parts of the plugin are read only when the plugin is actually used.
+* Format: Binary shared library files in the machine format of the host
+  architecture.
+
+State Tracking Files
+""""""""""""""""""""
+* Location and file name(s): ``VUSER_HOME/stateA.B.C.txt`` where ``A``,
+  ``B`` and ``C`` form a VisIt_ version number.
+* Purpose: Holds a single ASCII integer indicating the number of times the
+  associated VisIt_ version has been run. This is to facilitate suppression of
+  the release notes and help upon the *first* run of a new version of VisIt_.
+* Written: Each time VisIt_ is started, the integer value in the associated
+  state tracking file is updated.
+* Read: Each time VisIt_ is started, the value in the associated state tracking
+  file is read.
+* Format: ASCII text
+
+Crash Recovery Files
+""""""""""""""""""""
+* Location and file name(s): ``VUSER_HOME/crash_recovery.$pid.session`` and
+  ``VUSER_HOME/crash_recovery.$pid.session.gui`` where ``$pid`` is the process
+  id of the VisIt_ viewer component.
+* Purpose: Hold the most recently saved last good state of VisIt_ prior
+  to a crash. Disabled if the preference
+  ``Periodically save a crash recovery file`` is unchecked in the
+  **Preferences Window**.
+* Written: Periodically from VisIt_ automatically.
+* Read: When user starts VisIt_ and answers ``yes`` when queried whether to
+  start up from the most recent crash recovery file or when user explicitly
+  specifies the crash recovery file as an argument to the ``-sessionfile``
+  command-line :ref:`startup option <StartupOptions>`.
+* Format: ASCII `XML <https://en.wikipedia.org/wiki/XML>`_, same as any
+  other VisIt_ :ref:`session files <Session files>`.
+
+Files In Other Locations
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+There are several other kinds of files VisIt_ reads and writes to locations
+other than ``VUSER_HOME``. These are breifly described here.
+
+VisIt Debug Log (``.vlog``) Files
+"""""""""""""""""""""""""""""""""
+* Location and file name(s): On Windows, these files are always located in
+  ``VUSER_HOME``. On Unix/OSX the location of these files depends on whether
```

Comment:
The way its written, it sounds like client-server on windows also writes server files in VUSER_HOME on the client ("always located in VUSER_HOME"). 

Perhaps move the first sentence to the end of the paragraph, and modify "these files" to something like "local run only log files and client log files in client-server mode".